### PR TITLE
Cook/101 refacto timer

### PR DESCRIPTION
**Pull request by osvegn**
Pull request link to the issue: #101.
It updates timer to sleep.
It also removes _workInProgress variable and remove getStatus function.

### CODE REVIEW

File: src/Core/Cook/Cook.cpp [hunk 1]
```diff
@@ -10,45 +10,37 @@
 #include "Cook.hpp"
 #include <thread>
 
+extern "C" {
```
Comment (Cavonstavant) [inline src/Core/Cook/Cook.cpp:13]: useless

**Review by Cavonstavant — APPROVED** [review]
LGTM!

File: src/Core/Cook/Cook.cpp (hunk 1 above)
Comment (osvegn) [reply src/Core/Cook/Cook.cpp:13]: unistd.h is in C, so it need to be encalpsulate

**Review by 31Nathan — APPROVED** [review]
LGTM!

**Pull request merged**